README.md

```diff
@@ -8,18 +8,18 @@ It's a port of [the pizzapi node.js module](https://github.com/RIAEvangelist/nod
 
 First construct a `Customer` object and set the customer's address.
 ```python
->>> customer = Customer('Barack', 'Obama', 'barack@whitehouse.gov', '2024561111')
->>> customer.set_address('700 Pennsylvania Avenue NW', 'Washington', 'DC', '20408')
+>>> customer = Customer('Barack', 'Obama', 'barack@whitehouse.gov', '2024561111', '700 Pennsylvania Avenue NW, Washington, DC, 20408')
+>>> address = Address(*customer.address.split(','))
 ```
 
 Then, find a store that will deliver to the address.
 ```python
->>> store = find_closest_store(customer.address)
+>>> store = Address.closest_store(address)
```

Comment:
address.closest_store() would make more sense

Reply:
hmm yep.  i agree.  not sure what i was doing at that point.  i still need to do some clean up since my main goal was to just get it working.  ill pick it up again in the morning.
thanks for your feedback.